ballerina/metadata_types.bal

```diff
@@ -52,11 +67,43 @@ public type EntityFieldMetadata record {|
     RelationMetadata relation;
 |};
 
+# Represents the metadata associated with a field of an entity.
+# Only used by the generated persist clients and `persist:SQLClient`.
+#
+public type SheetFieldMetadata SimpleSheetFieldMetadata;//|SheetEntityFieldMetadata;
```

Comment:
```suggestion
public type SheetFieldMetadata SimpleSheetFieldMetadata;
```

Reply:
updated in [4346c39](https://github.com/ballerina-platform/module-ballerina-persist/pull/201/commits/4346c3959a465aeb05de0872b94ea1056a715a9a)